src/nvim/ops.c

```diff
@@ -4654,13 +4697,34 @@ char_u get_reg_type(int regname, long *reglen)
   yankreg_T *reg = get_yank_register(regname, YREG_PASTE);
 
   if (reg->y_array != NULL) {
-    if (reglen != NULL && reg->y_type == MBLOCK)
-      *reglen = reg->y_width;
+    if (reg_width != NULL && reg->y_type == MBLOCK)
+      *reg_width = reg->y_width;
     return reg->y_type;
   }
   return MAUTO;
 }
 
+void format_reg_type(char_u reg_type, colnr_T reg_width, char* buf, size_t buf_len)
```

Comment:
Missing documentation and function attributes. Also if signedness matter use `uint8_t` (otherwise `char`), `char_u` should be avoided and will be removed in the far future.